src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+
+    private Long id;

```

Comment:
The 'id' field should not be part of a DTO used for creating new entities, as the ID is typically generated by the database. Consider removing this field.